managed/services/server/server.go

```diff
@@ -895,7 +897,15 @@ func isAgentsStateUpdateNeeded(mr *serverpb.MetricsResolutions) bool {
 	return true
 }
 
-// check interfaces
+// check interfaces.
 var (
 	_ serverpb.ServerServer = (*Server)(nil)
 )
+
+func canUpdateDurationSetting(valueToBeSet, valueFromEnvironmentVariable time.Duration) bool {
+	if valueToBeSet == 0 || valueFromEnvironmentVariable == 0 || valueToBeSet == valueFromEnvironmentVariable {
+		return true
+	}
+
+	return false
```

Comment:
Thank you for your feedback. Since the PR was already approved I decided to merge. We can address this in https://github.com/percona/pmm/pull/989

Re the OR vs AND - Shashank suggested we switch to OR and I think it's easier to understand as well.
Re the function name - I prefer to use a positive name for functions, purely because if you negate the function you get `not(cannotUpdate)` which I think is harder to comprehend than `not(canUpdate)`